frontend/src/components/editor/cell/code/cell-editor.tsx

```diff
@@ -359,6 +361,17 @@ const CellEditorInternal = ({
     };
   }, [editorViewRef]);
 
+  // Completely hide the editor & icons if it's markdown and hidden. If there is output, we show.
+  const showHideButton =
```

Comment:
wonder if these should be memoized..

Reply:
These are just boolean operations, should be super cheap.